src/components/DefinitionList/index.ts

```diff
@@ -0,0 +1,2 @@
+export {DefinitionList} from './DefinitionList';
+export type {DefinitionListProps, DefinitionListItem} from './types';
```

Comment:
DefinitionListItem should be a component, not a type. Let's also reexport `DefinitionListItemProps`